src/tfr_msgs/action/Navigation.action

```diff
@@ -1,22 +1,16 @@
 #goal msg
 Header header
-uint8 location_code
+uint8 location_code 
```

Comment:
Added whitespace?!

Reply:
yeah that's weird, I don't know why that happened.